doc/user/content/release-notes.md

```diff
@@ -107,29 +107,31 @@ These changes are present in [unstable builds](/versions/#unstable-builds) and
 are slated for inclusion in the next stable release. There may be additional
 changes that have not yet been documented.
 
+{{< comment >}}
+Only add new release notes above this line.
+
+The presence of this comment ensures that PRs that are alive across a release
+boundary don't silently merge their release notes into the wrong place.
+{{</ comment >}}
+
+{{% version-header v0.16.0 %}}
+
 - **Breaking change.** Return an error when [`extract`](/sql/functions/extract/)
-  is called with a [`time`] value but a date-related field (e.g., `YEAR`).
+  is called with a [`time`] value but a date-related field (e.g., `YEAR`) {{% gh 9839 %}}.
 
   Previous versions of Materialize would incorrectly return `0` in these cases.
   The new behavior matches PostgreSQL.
 
 - **Breaking change.** Disallow the string `'sNaN'` (in any casing) as a valid
   [`numeric`] value.
 
-- Add the `array_remove` and `list_remove` functions.
-
-- Allow `SET NAMES ____` special case as per: https://www.postgresql.org/docs/9.1/sql-set.html.
-  Errors if the value is not `"UTF8"`.
+- Add the [`array_remove`](https://materialize.com/docs/sql/functions/#array-func) and [`list_remove`](https://materialize.com/docs/sql/functions/#array-func) functions.
 
-- Correctly parse `SET SCHEMA ____` special case as per: https://www.postgresql.org/docs/9.1/sql-set.html,
-  but it remains unsettable.
+- Support the `NAMES` and `SCHEMA` aliases in `SET` to [match the behavior](https://www.postgresql.org/docs/9.1/sql-set.html) of PostgreSQL, under the following conditions:
```

Comment:
The `SET` changes were originally two separate notes. Condensed them into a single note to make things less verbose.

Reply:
I'd simplify even further and say "Support the [special PostgreSQL syntax `SET NAMES` and `SET SCHEMA`](https://www.postgresql.org/docs/14/sql-set.html#id-1.9.3.173.6)  for setting the `client_encoding` and `search_path` parameters, respectively"? The detail about the semantics for these ops belongs in dedicated documentation on `SET` and `SHOW`... which sadly we don't have yet. Sigh. Regardless I don't think the release notes should be in the business of getting into that level of detail.

Also can we update the link to https://www.postgresql.org/docs/14/sql-set.html#id-1.9.3.173.6? That's the current version of PostgreSQL and links to a slightly more specific section heading.